climada/hazard/low_flow.py

```diff
@@ -203,127 +203,189 @@ def set_from_nc(self, input_dir=None, centroids=None, countries=[], reg=None,
             centr_handling = 'full_hazard'
 
         # read data and call preprocessing routine:
-        self.data, centroids_import = _data_preprocessing_percentile(
+        self.data_df, centroids_import = data_preprocessing_percentile(
             percentile, yearrange, yearrange_ref, input_dir, gh_model, cl_model,
             scenario, scenario_ref, soc, soc_ref, fn_str_var, bbox, min_days_per_month,
-            keep_dis_data, yearchunks)
+            keep_dis_data, yearchunks, mask_threshold)
 
         if centr_handling == 'full_hazard':
             centroids = centroids_import
         self.identify_clusters()
 
-        # sum "dis" (days per month below threshold) per pixel and cluster_id
-        # and write to hazard.intensiy
+        self.make_climada_hazard(centroids, min_intensity, min_number_cells,
+                                 yearrange, yearrange_ref, gh_model, cl_model,
+                                 scenario, scenario_ref, soc, soc_ref, fn_str_var, keep_dis_data)
+
+    def make_climada_hazard(self, centroids=None, min_intensity=1, min_number_cells=1,
```

Comment:
This function name could be more describtive "set_from_cluster"? or "make_hazard_from_cluster"?

Reply:
that's a good suggestion, will be changed to `set_intensity_from_clusters`